modules/codemod/lib/v7/recategorizeIconButtons.ts

```diff
@@ -0,0 +1,192 @@
+        const isInverseVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'inverse';
+        const isInverseFilledVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'inverseFilled';
+
+        if (isCircleVariant) {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'TertiaryButton';
+        } else if (isCircleFilledVariant) {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'SecondaryButton';
+        } else if (isInverseVariant) {
+          buttonType = 'TertiaryButton';
+        } else if (isInverseFilledVariant) {
+          buttonType = 'SecondaryButton';
+        } else {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'TertiaryButton';
+        }
+
+        updateJSXTag(nodePath, buttonType);
+        requiredImportSpecifiers.push(buttonType);
+      }
+    });
+
+  // Find all instances of IconButton within a style function
+  // const StyledIconButton = styled(IconButton) gets renamed to
+  // const StyledIconButton = styled(CorrectButtonMapping)
+  root.find(j.VariableDeclarator).forEach(nodePath => {
+    if (
+      nodePath.value.init?.type === 'CallExpression' &&
+      nodePath.value.init.callee.type === 'CallExpression' &&
+      nodePath.value.init.callee.arguments[0].type === 'Identifier'
+    ) {
+      console.log(nodePath.value.init.callee.arguments[0].name);
+      console.log(buttonType);
+
+      nodePath.value.init.callee.arguments[0].name = buttonType;
+      requiredImportSpecifiers.push(buttonType);
+    }
+  });
+
+  /**
+   * Remove old imports: `IconButton`
+   * Add new required imports
+   */
+  const buttonImports = root.find(j.ImportDeclaration, {
+    source: {value: '@workday/canvas-kit-react/button'},
+  });
+
+  const buttonImportFromMain = root.find(j.ImportDeclaration, {
+    source: {value: '@workday/canvas-kit-react'},
+  });
+  console.log(buttonImportFromMain);
+
+  if (!buttonImports.length) {
+    // Add new specifiers to a new import
+    const allImports = root.find(j.ImportDeclaration);
+
+    const lastImport = allImports.at(allImports.length);
+    if (lastImport) {
+      console.log(requiredImportSpecifiers);
+
+      lastImport.insertAfter(
+        j.importDeclaration(
+          requiredImportSpecifiers.map(specifier => j.importSpecifier(j.identifier(specifier))),
+          j.stringLiteral('@workday/canvas-kit-react/button')
+        )
+      );
+    }
+  } else {
+    buttonImports.forEach(({node}) => {
+      const specifiersToRemove = ['IconButton'];
+      console.log(node);
+
+      // Remove old specifiers
+      if (
+        typeof node.source.value === 'string' &&
+        node.source.value === '@workday/canvas-kit-react/button'
```

Comment:
```suggestion
        node.source.value.includes('@workday/canvas-kit-react')
```

It will apply all changes to IconButton imported from button package or main package